controlsfx-samples/src/main/java/org/controlsfx/samples/tableview2/HelloFilteredTableView.java

```diff
@@ -0,0 +1,776 @@
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.CheckBox;
+import javafx.scene.control.CheckMenuItem;
+import javafx.scene.control.ContextMenu;
+import javafx.scene.control.Label;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.Menu;
+import javafx.scene.control.MenuItem;
+import javafx.scene.control.RadioButton;
+import javafx.scene.control.SelectionMode;
+import javafx.scene.control.TableCell;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.TitledPane;
+import javafx.scene.control.ToggleGroup;
+import javafx.scene.control.cell.CheckBoxTableCell;
+import javafx.scene.control.cell.TextFieldTableCell;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.Circle;
+import javafx.stage.Stage;
+import javafx.util.StringConverter;
+import org.controlsfx.ControlsFXSample;
+import org.controlsfx.control.action.ActionUtils;
+import org.controlsfx.control.tableview2.FilteredTableColumn;
+import org.controlsfx.control.tableview2.FilteredTableView;
+import org.controlsfx.control.tableview2.actions.ColumnFixAction;
+import org.controlsfx.control.tableview2.actions.RowFixAction;
+import org.controlsfx.control.tableview2.cell.ComboBox2TableCell;
+import org.controlsfx.control.tableview2.cell.TextField2TableCell;
+import org.controlsfx.control.tableview2.filter.filtereditor.SouthFilter;
+import org.controlsfx.control.tableview2.filter.popupfilter.PopupFilter;
+import org.controlsfx.control.tableview2.filter.popupfilter.PopupNumberFilter;
+import org.controlsfx.control.tableview2.filter.popupfilter.PopupStringFilter;
+import org.controlsfx.samples.Utils;
+
+/**
+ *
+ * Build the UI and launch the Application
+ */
+public class HelloFilteredTableView extends ControlsFXSample {
+
+    private final ObservableList<Person> data = generateData(100);
+    private FilteredTableViewSample table;
+    private StackPane centerPane;
+    private BooleanProperty southVisible = new SimpleBooleanProperty();
+
+    public static void main(String[] args) {
+        launch(args);
+    }
+
+    @Override
+    public String getSampleName() {
+        return "FilteredTableView";
+    }
+
+    @Override
+    public String getSampleDescription() {
+        return "The FilteredTableView is a subclass of TableView2,"
+                + " an advanced JavaFX TableView control that provides extended filtering options. It can"
+                + " be used as drop-in replacement control for the existing TableView, and provides"
+                + " different functionalities and use cases like row and column fixing, row header and"
+                + " filtering editors.";
+    }
+
+    @Override
+    public String getControlStylesheetURL() {
+        return "/org/controlsfx/samples/tableview2.css";
```

Comment:
This URL should return the CSS used by the Control and not the sample. Please return the following:

```
return "/org/controlsfx/control/tableview2/tableview2.css";
```

Also, update the `controlsfx/src/main/java/module-info.java` to include the following so that FXSampler can read the css:

```
opens org.controlsfx.control.tableview2 to org.controlsfx.fxsampler;
```

Reply:
done